client/js/index.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
```

Comment:
delete this file?

Reply:
using it to test, will delete just before merge